superset-frontend/cypress-base/cypress/integration/dashboard/edit_properties.test.ts

```diff
@@ -97,10 +97,6 @@ describe('Dashboard edit action', () => {
       .get('[data-test="dashboard-title-input"]')
       .type(`{selectall}{backspace}${dashboardTitle}`);
 
-    cy.wait('@dashboardGet').then(() => {
-      selectColorScheme('d3Category20b');
-    });
-
```

Comment:
This was here because the color scheme was considered to e mandatory. This is not the case any longer